### DISCUSSION TITLE
Wrap a non-helm manifest with Helmfile

### DISCUSSION THREAD

**Discussion by kjq** [helmfile/helmfile#864] [Questions]
Is there a way to at least pull in a regular manifest that is _not_ a Helm chart or Helmfile and use Helmfile to manage its deployment?

I would like to keep all Kubernetes resources under Helmfile but with some third-party resources they either are just a manifest or operators - both without Helm charts.

**Comment by yxxhero** [comment]
@kjq yeah. you can use a dir as a chart.
